HunterPie.Core/Client/Configuration/ClientConfig.cs

```diff
@@ -37,6 +37,9 @@ public class ClientConfig : ISettings
         [SettingField("SEAMLESS_SHUTDOWN_STRING")]
         public Observable<bool> EnableSeamlessShutdown { get; set; } = false;
 
+        [SettingField("ENABLE_NATIVE_MODULE_STRING", availableGames: GameProcess.MonsterHunterWorld)]
+        public Observable<bool> EnableNativeModule { get; set; } = false;
```

Comment:
I feel like this can be enabled by default, since most MHW players has Stracker's and CRCBypass installed

Reply:
I'm not sure. But we might as well add a line of hint in the logs in case there is someone not installing these mods.

Reply:
Sure, you can check MHW's folder for the `hid.dll` and `loader.dll`, if they exist, then the user has Stracker's installed. And since CRCBypass is a requirement for Stracker's then we can assume they also have CRC Bypass, otherwise their game would crash without it.

Reply:
Probably. I think I can try to do this in another PR, though.

I've changed default to `true`.